third_party/move/move-prover/boogie-backend/src/bytecode_translator.rs

```diff
@@ -1492,6 +1740,36 @@ impl<'env> FunctionTranslator<'env> {
                             field_sel,
                         );
                     },
+                    BorrowVariantField(mid, sid, variants, inst, field_offset) => {
+                        let inst = &self.inst_slice(inst);
```

Comment:
Why do we need to use `inst` here when we didn't in `BorrowField(mid, ...)` above?

Reply:
We need `inst` here because we need to generate struct variant name below, which is different from `BorrowField` or `GetField`. And since `inst_slice` is called every time `inst` is used, I also call it here.